src/app/shared/models/history-log.model.ts

```diff
@@ -0,0 +1,53 @@
+import {User} from "./user.model";
+
+export interface Provider {
```

Comment:
it seems that we already have a similar name for the interface,
let`s try to use another name